### DISCUSSION TITLE
Change appendix and annex numbering to letters

### DISCUSSION THREAD

**Discussion by armm1009** [joaomlourenco/novathesis#464] [General]
Good morning,

Is it possible to change appendix and annex numbering to letters?

Instead of I, II, III, IV, etc. I wanted A, B, C, D,....

Thank you in advance for your time.

**Comment by joaomlourenco** [comment]
By default, in the template, appendices use letters and annexes use roman numbering.
What do you really want?  That both appendices and annexes use letters?  Do you want Annexes to restart the letters or want them to continue from appendices?
Please be clear…. thank you.
